doc/sphinx/src/using-closures.rst

```diff
@@ -167,6 +398,23 @@ objects for each equation of state, similar to the vector API for a
 given EOS. Here the indexers/vectors are not over cells, but
 materials.
 
+.. warning::
+
+  It bears repeating: **both the volume fractions and densities act as inputs
+  and outputs**. They are used to define the internal :math:`\overline
```

Comment:
An internal variable is not necessarily constant, so add constant somewhere to underscore that THIS is how the two input variables are coupled.

Reply:
fixed